distribution/docs/src/main/resources/_contents/_configuring/configuring-from-the-command-console-contents.adoc

```diff
@@ -0,0 +1,112 @@
+[installed  ] [${project.version}  ] security-sts-server                      security-services-app-${project.version} DDF Security STS.
+[installed  ] [${project.version}  ] security-sts-realm                       security-services-app-${project.version} DDF Security STS Realm.
+[uninstalled] [${project.version}  ] security-sts-ldaplogin                   security-services-app-${project.version} DDF Security STS JAAS LDAP Login.
+[uninstalled] [${project.version}  ] security-sts-ldapclaimshandler           security-services-app-${project.version} Retrieves claims attributes from an LDAP store.
+----
+. Check the bundle status to verify the service is started. +
+`${branding-lowercase}${at-symbol}local>list`
+
+The console output should show an entry similar to the following:
+----
+[ 117] [Active     ] [            ] [Started] [   75] ${ddf-branding} :: Catalog :: Source :: Dummy (<version>)
+----
+
+===== Uninstall Features
+
+. Check the feature list to verify the feature is installed properly. +
+`${branding-lowercase}${at-symbol}local>feature:list`
+
+----
+State         Version          Name                          Repository  		   Description
+[installed  ] [${project.version}         ] ${ddf-branding-lowercase}-core                      ddf-${project.version}
+[uninstalled] [${project.version}         ] ${ddf-branding-lowercase}-sts                       ddf-${project.version}
+[installed  ] [${project.version}         ] ${ddf-branding-lowercase}-security-common           ddf-${project.version}
+[installed  ] [${project.version}         ] ${ddf-branding-lowercase}-resource-impl             ddf-${project.version}
+[installed  ] [${project.version}         ] ${ddf-branding-lowercase}-source-dummy              ddf-${project.version}
+----
+
+. Uninstall the feature. +
+`${branding-lowercase}${at-symbol}local>feature:uninstall ${branding-lowercase}-source-dummy`
+
+[WARNING]
+====
+Dependencies that were auto-installed by the feature are not automatically uninstalled.
+====
+
+. Verify that the feature has uninstalled properly. +
+`${branding-lowercase}${at-symbol}local>feature:list`
+
+----
+State         Version          Name                          Repository  Description
+[installed  ] [${project.version}         ] ${ddf-branding-lowercase}-core                      ddf-${project.version}
+[uninstalled] [${project.version}         ] ${ddf-branding-lowercase}-sts                       ddf-${project.version}
+[installed  ] [${project.version}         ] ${ddf-branding-lowercase}-security-common           ddf-${project.version}
+[installed  ] [${project.version}         ] ${ddf-branding-lowercase}-resource-impl             ddf-${project.version}
+[uninstalled] [${project.version}         ] ${ddf-branding-lowercase}-source-dummy              ddf-${project.version}
+----
+
+==== Configuring HTTP to HTTPS Proxy From the ${command-console}
```

Comment:
Like the Admin Console, should we have a more general section related to configuring some of the main optional features and include the http proxy as a sub-section?